packer/edge-vars.json

```diff
@@ -0,0 +1,3 @@
+{
+    "iso_file": "linuxmint-{{user `version`}}-cinnamon-64bit-edge.iso"
```

Comment:
Do we really want to introduce the edge builds into this? I don't even think we can run successfully on them at this point, and VirtualBox is unlikely to give us the exotic hardware its intended for.

Reply:
A quick run worked for me using edge but yeah it probably doesn't provide much value. Included it as an example of being able to reference `version` in another file but it's not particularly useful. I'll push another change later today with it removed.